pkgs/development/libraries/cassandra-cpp/default.nix

```diff
@@ -0,0 +1,34 @@
+{ pkgs, examples ? false}: pkgs.stdenv.mkDerivation rec {
```

Comment:
you should not take `pkgs` as argument but all the members you need: `stdenv, fetchgit, ...`

Reply:
Roger that. Fixed.